src/Dev/Deprecation.php

```diff
@@ -179,7 +179,7 @@ public static function get_enabled()
         // noop
     }
 
-    private static function get_is_enabled(): bool
+    public static function get_is_enabled(): bool
```

Comment:
This needs to be public now as we use this for skipping unit test of deprecated code, i.e.
```php
if (Deprecation::get_is_enabled()) {
    $this->markTestSkipped('Test calls deprecated code');
}
```